man/systemd-nspawn.xml

```diff
@@ -477,6 +466,23 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>-U</option></term>
+
+        <listitem><para>If the kernel supports the user namespaces feature, equivalent to
+        <option>--private-users=pick --private-users-chown</option>, otherwise equivalent to
+        <option>--private-users=no</option>.</para>
+
+        <para>Note that <option>-U</option> is the default if the <filename>systemd-nspawn@.service</filename> template unit
+        file is used.</para>
+
+        <para>Note: it is possible to undo the effect of <option>--private-users-chown</option> (or <option>-U</option>)
+        by redoing the operation with the first UID of 0:</para>
```

Comment:
BTW: I think it would be great to run something
```
systemd-nspawn --uidshift ... --dont-spawn-container
```
Similar to [`fuidshift`](https://github.com/systemd/systemd/issues/4078#issuecomment-249446811) :-)